cfgov/v1/migrations/0086_auto_20160520_1351.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
```

Comment:
Delete the migrations, then run `makemigrations` again. It will generate the one file that encompasses all the changes. Also, if you've already run these migrations locally, you can run `cfgov/manage.py migrate v1 --fake` after you make the one migration file to register that migration as already done.